bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
Max, 

It's very simple... xmlrpc returns xml. buglist returns html. One is used by python to process and turn into JIRAs. The other is used to simply see the pretty HTML list of bugs.

So, either you should be echoing the XML query to the console:

xmlrpc.cgi?last_change_time=2016-01-27%2010%3A00&status_whiteboard=RHT

or the equivalent HTML query:

buglist.cgi?chfieldfrom=2016-01-27%2010%3A00&status_whiteboard=RHT

But buglist.cgi doesn't support last_change_time. It's not a field that the cgi script understands. That's why when I showed examples above, in one case you get a filtered result (1 bug found) and in the other you get all 61 (filter is ignored).

I don't know how else to explain this. xmlrpc API uses one field name; buglist API uses a different one.

To explain that further:

* XMLRPC API spec: https://www.bugzilla.org/docs/4.4/en/html/api/Bugzilla/WebService/Bug.html#get (search for last_change_time )
* buglist.cgi example using chfieldfrom: https://bugzilla.mozilla.org/show_bug.cgi?id=649757
* https://bugs.eclipse.org/bugs/query.cgi?chfieldfrom=2016-01-21T23%3A00%3A00&query_format=advanced&status_whiteboard=RHT (note the Search By Change History > between: ... value. When you hit the "search" button at the bottom of the query form, you can then see the resulting URL uses chfieldfrom, not last_change_time)

---

IMHO, putting a URL in the console which doesn't work when you copy and paste it into a browser makes it not very useful. So, if I ran the circus, I'd put a URL in the console that represented the same HTML results with buglist.cgi that is returned in XML via xmlrpc.cgi to python directly. The purpose is not to show the exact query passed to the server, but to allow a human-readable query result that equates to what is being processed by the script.

If that still doesn't compute, then I give up.